application-tour-ui/src/main/resources/TourCode/StepsListing.xml

```diff
@@ -20,24 +20,21 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.3" reference="TourCode.StepsListing" locale="">
```

Comment:
Note: Updates on the XAR format of a document should be best committed separately, so that they don`t mix with the current changes of the document. In other words, export the document you have modified, but with a version that does not have your changes and commit only the XAR format updates. Then come back in your wiki to the version that has you changes and export and commit that. This way, only your actual changes get exposed in the commit and are not mixed with other plumbing stuff like tags reordering, formatting, etc.